console/services/app_config/volume_service.py

```diff
@@ -190,13 +190,9 @@ def check_volume_path(self, service, volume_path, local_path=[]):
                 volume_path_win = True
             if not volume_path.startswith("/") and not volume_path_win:
                 raise ServiceHandleException(msg="path error", msg_show="路径仅支持linux和windows")
-            if volume_path in self.SYSDIRS:
```

Comment:
1. The validation logic for the system directory cannot be removed directly, but the directory option can be optimized.
2. Is the installation process not handled?